components/TextFields/src/MDCTextField.m

```diff
@@ -226,6 +228,25 @@ - (void)updateBorder {
   self.borderView.borderPath = self.borderPath;
 }
 
+#pragma mark - Input Layout Strut Implementation
+
+- (void)setupInputLayoutStrut {
+  self.inputLayoutStrut = [[UILabel alloc] initWithFrame:CGRectZero];
+  //self.inputLayoutStrut.alpha = 0.f;
+  [self addSubview:self.inputLayoutStrut];
+
+  self.inputLayoutStrut.numberOfLines = 1;
+  self.inputLayoutStrut.alpha = 0.f;
```

Comment:
Do we want to set:
self.inputLayoutStruct.hidden = YES

Reply:
Yeah. This is one of those times where everything still works when hidden.

Reply:
Done.